style.css

```diff
@@ -11,24 +11,45 @@
 
 body {
   /* I prefer sans-serif (without curls) so I'm changing it haha */
-  font-family: Helvetica, sans-serif;
+	/* lets see what Arial looks like*/
+  font-family: Arial;
+background-color: #afeeee;
```

Comment:
You should always make an effort to correctly indent your code. It makes it much easier to read later on and others can naturally identify what is a selector and what is a property based on the indentation.